packages/features/ee/teams/pages/team-members-view.tsx

```diff
@@ -25,6 +25,18 @@ const MembersView = () => {
     onError: () => {
       router.push("/settings");
     },
+    enabled: !!router.query.id,
```

Comment:
Adding this will only run the query when `useRouter` has time to parse the query params

@hariombalhara

Reply:
Why not use router.isReady ?

Reply:
Good call